scalding-commons/src/main/scala/com/twitter/scalding/commons/source/LzoCodecSource.scala

```diff
@@ -30,6 +30,6 @@ object LzoCodecSource {
       val hdfsPaths = paths
       val localPath = { assert(paths.size == 1, "Cannot use multiple input files on local mode"); paths(0) }
       val boxed = Externalizer(passedInjection)
-      override def injection = boxed.get
+      override lazy val injection = boxed.get
```

Comment:
Externalizer is already doing the equivalent of lazy, isn't it? Do we need it twice?